### PR TITLE
[SPARKTA-337] JDBC Output

### PR DESCRIPTION
**Pull request by compae**
#### Description

New output for save the aggregated data into SQL Database with JDBC connector.

Is mandatory define properties like:

"url"="jdbc:mysql://localhost:3306/hadoopdb"

Optionals:

"driver" = ...
"user" = ...
"password" = ...

#### Test
All passed.

### CODE REVIEW

File: plugins/output-jdbc/src/main/scala/com/stratio/sparkta/plugin/output/jdbc/JdbcOutput.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import scala.collection.JavaConversions._
+
+class JdbcOutput(keyName: String,
+                 version: Option[Int],
+                 properties: Map[String, JSerializable],
+                 operationTypes: Option[Map[String, (WriteOp, TypeOp)]],
+                 bcSchema: Option[Seq[TableSchema]])
+  extends Output(keyName, version, properties, operationTypes, bcSchema) {
+
+  val url = properties.getString("url", None).getOrElse(throw new IllegalArgumentException("url must be provided"))
```
Comment (sgomezg) [inline plugins/output-jdbc/src/main/scala/com/stratio/sparkta/plugin/output/jdbc/JdbcOutput.scala]: I suggest:
```
require(properties.getString("url").isDefined, "url must be provided")
val url = properties.getString("url").get
```

**Comment by Gschiavon** [comment]
This PR should include documentation in the website to explain the new output!
Good job.

**Comment by compae** [comment]
Thanks @sgomezg, I change the throw for require.

**Comment by danielcsant** [comment]
Add template

https://github.com/Stratio/Workflow/blob/master/web/src/main/resources/data-templates/fake_data/items.json

**Comment by acemilio** [comment]
In order to maintain the current front-end version, you should update output.json and en-US.json files correctly.

**Comment by compae** [comment]
Updated web project.

**Comment by danielcsant** [comment]
Please add documentation:
https://stratio.atlassian.net/wiki/display/SPARKTA/7.+Reference+Guide

**Pull request merged**